canonical-web-and-design/vanilla-framework
# Move global nav styles from example to scss file

## Done

Moved global nav css from the documentation example to its own scss file.
- includes a drive by fix for the skip link standalone example that was added recently

Fixes #3109 

## QA
- Open [demo](https://vanilla-framework-3852.demos.haus/docs/layouts/documentation#example)
- See that the markup in the code snippet example doesn't contain a really long line of CSS at the top, but that the global nav renders correctly in the example.
  - Compare to [live](https://vanillaframework.io/docs/layouts/documentation#example)

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
